src/plugins/irc-events/error.js

```diff
@@ -10,6 +10,16 @@ module.exports = function(irc, network) {
 
 		if (data.reason) {
 			text = data.reason + " (" + text + ")";
+			if (data.channel) {
+				text = data.channel + ": " + text;
```

Comment:
Can you move `data.channel` outside of any other if so that `+=` can be used in text formatting.